patches/master_patch.patch

```diff
@@ -1842,6 +1842,829 @@ index b73f65168086189aed021cc82ffef5fde1f2d061..aa3f97fa57a3cb5616cd3811a34d0698
++++ b/net/http/transport_security_state.cc
+@@ -758,9 +758,9 @@ TransportSecurityState::TransportSecurityState()
+ // Static pinning is only enabled for official builds to make sure that
+ // others don't end up with pins that cannot be easily updated.
+ #if !defined(GOOGLE_CHROME_BUILD) || defined(OS_ANDROID) || defined(OS_IOS)
+-  enable_static_pins_ = false;
+   enable_static_expect_ct_ = false;
+ #endif
++  enable_static_pins_ = false;
+   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
+ }
+ 
+diff --git a/net/http/transport_security_state_static.json b/net/http/transport_security_state_static.json
+index 698980a564bc1e72bdc3967244fcba0dcd610af5..24b2f6634bfc0d1e6883a854cedf7c32af76d5a5 100644
+--- a/net/http/transport_security_state_static.json
++++ b/net/http/transport_security_state_static.json
+@@ -55,6 +55,19 @@
+         "TestSPKI"
+       ]
+     },
++    {
++      "name": "brave",
++      "static_spki_hashes": [
++        "DigiCertGlobalRoot",
++        "DigiCertEVRoot",
++        "DigiCertAssuredIDRoot",
++        "GlobalSignRootCA",
++        "GlobalSignRootCA_R2",
++        "GlobalSignRootCA_R3",
++        "LetsEncryptAuthorityPrimary_X1_X3",
++        "LetsEncryptAuthorityBackup_X2_X4"
++     ]
++   },
+     {
+       "name": "google",
+       "static_spki_hashes": [
+@@ -246,16 +259,24 @@
+ 
+   "entries": [
+     // Dummy entries to test certificate pinning and expect-CT.
+-    { "name": "pinningtest.appspot.com", "include_subdomains": true, "pins": "test" },
+-    { "name": "pinning-test.badssl.com", "include_subdomains": true, "pins": "test" },
+-    { "name": "preloaded-expect-ct.badssl.com", "expect_ct": true, "expect_ct_report_uri": "https://clients3.google.com/ct_upload" },
+     { "name": "preloaded-expect-staple.badssl.com", "expect_staple": true, "expect_staple_report_uri": "https://report.badssl.com/expect-staple" },
+     { "name": "preloaded-expect-staple-include-subdomains.badssl.com", "expect_staple": true, "expect_staple_report_uri": "https://report.badssl.com/expect-staple", "include_subdomains_for_expect_staple": true },
+ 
++    //Brave
++    { "name": "ledger.mercury.basicattentiontoken.org", "mode": "force-https", "pins": "brave"},
++    { "name": "ledger-staging.mercury.basicattentiontoken.org", "mode": "force-https", "pins": "brave"},
++    { "name": "balance.mercury.basicattentiontoken.org", "mode": "force-https", "pins": "brave"},
++    { "name": "balance-staging.mercury.basicattentiontoken.org", "mode": "force-https", "pins": "brave"},
++    { "name": "laptop-updates.brave.com", "mode": "force-https", "pins": "brave"},
++    { "name": "sync.brave.com", "mode": "force-https", "pins": "brave"},
++    { "name": "sync-staging.brave.com", "mode": "force-https", "pins": "brave"},
++    { "name": "brave-laptop-updates.global.ssl.fastly.net", "mode": "force-https", "pins": "brave"},
++    { "name": "brave-download.global.ssl.fastly.net", "mode": "force-https", "pins": "brave"},
```

Comment:
we might want to wait on https://github.com/brave/browser-laptop/issues/12301 so we can pin brave.com domains instead of fastly.net domains